### PR TITLE
TEL - Champions of Yomi

### LINKED ISSUE #3521: TEL - Champions of Yömi
image

### PR DESCRIPTION
**Pull request by tvdstaaij**
closes #3521

### CODE REVIEW

File: test/server/cards/10-TEL/ChampionsOfYomi.spec.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                    defenders: [this.kakitaToshimoko]
+                });
+            });
+
+            it('should be put into play when triggered after losing', function() {
+                this.noMoreActions();
+                expect(this.player1).toBeAbleToSelect(this.championsOfYomi);
+                this.player1.clickCard(this.championsOfYomi);
+                expect(this.player1).toBeAbleToSelect(this.yojinNoShiro);
+                this.player1.clickCard(this.yojinNoShiro);
```
Comment (Matthew-Griffith) [inline test/server/cards/10-TEL/ChampionsOfYomi.spec.js]: after this you should check that the cost of the stronghold being bowed was paid.

Comment (tvdstaaij) [reply test/server/cards/10-TEL/ChampionsOfYomi.spec.js]: Good call, done.

File: test/server/cards/10-TEL/ChampionsOfYomi.spec.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
+fdescribe('Champions of Yomi', function() {
```
Comment (jeremylarner) [inline test/server/cards/10-TEL/ChampionsOfYomi.spec.js]: Need to remove `fdescribe`

Comment (tvdstaaij) [reply test/server/cards/10-TEL/ChampionsOfYomi.spec.js]: Right, I was in a hurry when committing the requested change this morning, sorry about that.

**Pull request merged**